MapboxVisionTests/VisionManagerTests.swift

```diff
@@ -0,0 +1,161 @@
+@testable import MapboxVision
+import XCTest
+
+class VisionManagerTests: XCTestCase {
+    var visionManager: VisionManager?
```

Comment:
I think it's safe to make it force-unwrapped to avoid unnecessary unwrapping.

Reply:
I prefer to use soft unwrappings